### PR TITLE
Docs: add the usage link for `coordinator_storage_quota_in_mb` of `azurerm_cosmosdb_postgresql_cluster`

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes https://github.com/hashicorp/terraform-provider-azurerm/issues/21761

### CODE REVIEW

File: website/docs/r/cosmosdb_postgresql_cluster.html.markdown [hunk 1]
```diff
@@ -43,6 +43,8 @@ The following arguments are supported:
 
 * `coordinator_storage_quota_in_mb` - (Required) The coordinator storage allowed for the Azure Cosmos DB for PostgreSQL Cluster. Possible values are `32768`, `65536`, `131072`, `262144`, `524288`, `1048576`, `2097152`, `4194304`, `8388608` and `16777216`.
 
+-> **NOTE:** For more information, please see the [product documentation](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/resources-compute)
```
Comment (WodansSon) [inline website/docs/r/cosmosdb_postgresql_cluster.html.markdown:46]: This is great, but when linking to a MS corporate asset can we remove the `local` from the docs embedded URL (e.g. `en-us`) to allow the web sites to geo-locate the requests origin and then select the most appropriate language to display the content in based on the requests origin? Also, if possible add an anchor to the link to the most relative section in the doc? I am not sure if there is one for this link, but just putting it out there as something to think about if it makes sense. 🙂 

```suggestion
-> **NOTE:** For more information, please see the [product documentation](https://learn.microsoft.com/azure/cosmos-db/postgresql/resources-compute)
```

**Review by WodansSon — CHANGES_REQUESTED** [review]
@neil-yechenwei , this mostly looks good and is 🔥 however there is a tiny thing that I feel should be addressed and then this should be GTG! 🚀

File: website/docs/r/cosmosdb_postgresql_cluster.html.markdown (hunk 1 above)
Comment (neil-yechenwei) [reply website/docs/r/cosmosdb_postgresql_cluster.html.markdown:46]: Thanks. Updated PR.

File: website/docs/r/cosmosdb_postgresql_cluster.html.markdown [hunk 2]
```diff
@@ -43,6 +43,8 @@ The following arguments are supported:
 
 * `coordinator_storage_quota_in_mb` - (Required) The coordinator storage allowed for the Azure Cosmos DB for PostgreSQL Cluster. Possible values are `32768`, `65536`, `131072`, `262144`, `524288`, `1048576`, `2097152`, `4194304`, `8388608` and `16777216`.
 
+-> **NOTE:** For more information, please see the [product documentation](https://learn.microsoft.com/azure/cosmos-db/postgresql/resources-compute)
```
Comment (tombuildsstuff) [inline website/docs/r/cosmosdb_postgresql_cluster.html.markdown:46]: This is unclear as to what we're linking too - more information on _what_ exactly?

A little more context would make this clearer, for example:

```suggestion
-> **NOTE:** More information on [the types of compute resources available for CosmosDB can be found in the product documentation](https://learn.microsoft.com/azure/cosmos-db/postgresql/resources-compute)
```

Comment (neil-yechenwei) [reply website/docs/r/cosmosdb_postgresql_cluster.html.markdown]: Thanks. Updated.

**Review by katbyte — APPROVED** [review]
LGTM 📜

**Pull request merged**